pgcli/packages/pgspecial.py

```diff
@@ -25,6 +25,26 @@ def parse_special_command(sql):
     verbose = '+' in command
     return (command.strip(), verbose, arg.strip())
 
+def list_schemas(cur, pattern, verbose):
+    """
+    Returns (rows, headers, status)
+    """
+    sql = """SELECT n.nspname AS "Name",
+    pg_catalog.pg_get_userbyid(n.nspowner) AS "Owner"
+    FROM pg_catalog.pg_namespace n WHERE n.nspname """
+
+    if pattern:
+        sql += "~ '^(" + pattern.replace('*', '.*') + ")$'"
```

Comment:
You might have to use [sql_name_pattern](https://github.com/amjith/pgcli/blob/master/pgcli/packages/pgspecial.py#L656) to do this.